CIP-0137/README.md

```diff
@@ -0,0 +1,581 @@
+---
+CIP: 137
+Title: Decentralized Message Queue
+Category: Network
+Status: Proposed
+Authors:
+    - Jean-Philippe Raynaud <jp.raynaud@gmail.com>
+    - Arnaud Bailly <arnaud.bailly@iohk.io>
+    - Marcin Szamotulski <marcin.szamotulski@iohk.io>
+    - Armando Santos <armando.santos@iohk.io>
+    - Neil Davies <neil.davies@iohk.io>
+    - Sebastian Nagel <sebastian.nagel@ncoding.at>
+Implementors: 
+    - Cardano Scaling team <https://github.com/cardano-scaling>
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/876
+Created: 2024-08-02
+License: Apache-2.0
+---
+
+## Abstract
+
+We propose to create a decentralized message diffusion protocol leveraging the Cardano network layer. This protocol allows to follow a topic based diffusion of messages from publishers to subscribers in a decentralized way.  
+
+The messages can be sent and received by nodes running in a non intrusive way side by side to the Cardano node in order to enable inter-nodes communications.
+
+In this way, we can significantly reduce the cost and effort required to build a decentralised network for message diffusion by using Cardano's established infrastructure, with limited impact on the performance and no impact on the security of the Cardano network.
+
+## Motivation: why is this CIP necessary?
+
+Mithril is a protocol based on a [Stake-based Threshold Multi-signature scheme](https://iohk.io/en/research/library/papers/mithril-stake-based-threshold-multisignatures/) which leverages the Cardano SPOs to certify data from the Cardano chain in a trustless way. Mithril is currently used in the Cardano ecosystem in order to enable fast bootstrapping of full nodes and enabling secure light wallets.
```

Comment:
I believe the motivation section could benefit from a more generalized framing before diving into the specific application of Mithril. As it currently stands, the document immediately presents Mithril as the driving example, which might confuse readers who are looking for a broader understanding of the need for a decentralized message queue.

I would suggest starting with a more generalized motivation that emphasizes the overarching need for decentralized message diffusion across various protocols in the Cardano ecosystem. The benefits of using the existing Cardano infrastructure for this purpose—such as efficiency, security, and reduced development overhead—should be highlighted first. Mithril can then be introduced as a specific use case, perhaps noting that it will be used as an illustrative example throughout the document.

While I understand Mithril will be a fundamental first user of this DMQ, it's important that the document doesn't feel too Mithril-centric.

Reply:
Thanks @perturbing for this feedback. I will adjust the motivation section in that manner shortly :+1:

Reply:
@perturbing I have updated the Motivation section to be less Mithril-centric in the commit https://github.com/cardano-scaling/CIPs/commit/7b304a559c4d59a7722782fcfb585f946fcdd5bc

Reply:
thanks @jpraynaud - I think https://github.com/cardano-foundation/CIPs/pull/876/commits/7b304a559c4d59a7722782fcfb585f946fcdd5bc does a great job of putting this CIP into context without being too specific with respect to Milthril.